### PR TITLE
Revise etcd task prerequisites to remove playground envs

### PR DESCRIPTION
**Pull request by tengqm**
The etcd operation and maintenance job is not supposed to run on the playground environments.

closes: #4479

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md [hunk 1]
```diff
@@ -14,7 +14,11 @@ weight: 270
 
 ## {{% heading "prerequisites" %}}
 
-{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+You need to have a Kubernetes cluster, and the kubectl command-line tool must
+be configured to communicate with your cluster. It is recommended to run this
+task on a cluster with at least two nodes that are not acting as control plane
+nodes . If you do not already have a cluster, you can create one by using
+[minikube](https://minikube.sigs.k8s.io/docs/tutorials/multi_node/)
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md:21]: (nit)
```suggestion
[minikube](https://minikube.sigs.k8s.io/docs/tutorials/multi_node/).
```

**Review by sftim — COMMENTED** [review]
Making this change sounds legit to me.

/lgtm
/approve 
/hold

@tengqm there's a nit, or you can unhold - the fix is not even worth a first issue

**Comment by Shubham82** [comment]
remove the hold label, as the above comment is addressed.

/unhold

**Comment by Shubham82** [comment]
Preview looks good: https://deploy-preview-43277--kubernetes-io-main-staging.netlify.app/docs/tasks/administer-cluster/configure-upgrade-etcd/#before-you-begin

/lgtm

**Pull request merged**